cashctrl_ledger/ledger.py

```diff
@@ -77,11 +209,19 @@ def delete_price():
         """
         raise NotImplementedError
 
-    def delete_vat_code():
+    def delete_vat_code(self, code: str):
         """
-        Not implemented yet
+        Deletes a VAT code from the remote CashCtrl account.
+
+        Parameters:
+            code (str): The VAT code name to be deleted.
         """
-        raise NotImplementedError
+        tax_rates = self._client.list_tax_rates()
+        to_delete = tax_rates.loc[tax_rates['name'] == code, 'id']
+
+        if len(to_delete) > 0:
+            delete_ids = ",".join(to_delete.astype(str))
+            self._client.post('tax/delete.json', {'ids': delete_ids})
```

Comment:
Let's add an else block, where we raise an exception "VAT code not found." if the given `code` does not match an existing one.